### ISSUE TITLE
Public Announcement

### ISSUE THREAD

**Issue by theapache64** [deepset-ai/COVID-QA#73]
It'd better if we announce this product to public in something like [producthunt](https://producthunt.com) etc or is it done somewhere?

**Comment by Timoeller** [comment]
This is a great idea. 
We think we should have two features until the project is ready for producthunt.
1. integrate a more intelligent matching of English questions through Bert based models.
2. change the UI to English

**Comment by sfakir** [comment]
@Timoeller  agree, or actually multilanguage, should not be too complicated and doable on the weekend.
